scripts/dataset_processing/tts/hui_acg/phonemizer_local.py

```diff
@@ -0,0 +1,78 @@
+"""
+The original JSON splits generated from `get_data.py` only contain graphemes inputs. We recommend adding phonemes as
```

Comment:
Also this will add phonemes to the manifests right? Are we sure we want to do that? And is this consistent with how we use or add phonemes in NeMo?

Reply:
It seems worth adding as an option, in addition to being able to do phonemizing on the fly.

I recall @yzhang123 and @ekmb have added some support for a specific format it should be in for the normalizer to pass it through untouched, could one of you confirm?

Reply:
@subhankar-ghosh yes. we will add new entry in the JSON. `{..."normalized_text": xyz, "is_phoneme": 1}`. We decided to use a mixer of grapheme/phoneme for German as suggested because German is kinda a phonetic language.

Reply:
The tokenizer using phonemes for German is different from what English used.